Sources/BraintreeVisaCheckout/BTVisaCheckoutAddress.swift

```diff
@@ -4,40 +4,41 @@ import Foundation
 import BraintreeCore
 #endif
 
-/// A model representing a user's address information returned by Visa Checkout.
-/// Parses and stores address values from a BTJSON object.
-@objc public class BTVisaCheckoutAddress: NSObject {
+
+// A class containing Visa Checkout information about the user's address.
+// Parses and stores address values from a `BTJSON` object.
```

Comment:
To match the rest of the SDK docstrings:
```suggestion
/// A class containing Visa Checkout information about the user's address.
/// Parses and stores address values from a `BTJSON` object.
```